src/index.ts

```diff
@@ -306,8 +307,16 @@ class Geckoboard {
     return new Dataset(schema);
   }
 
-  ping(): Promise<void> {
-    return Promise.resolve();
+  async ping(): Promise<void> {
+    const base64UserString = btoa(`${this.apiKey}:`);
+    const res = await fetch('https://api.geckoboard.com', {
+      headers: {
+        Authorization: `Basic ${base64UserString}`,
```

Comment:
can leave this until a later iteration, but we'll likely want a helper around the fetch or a helper to build to headers to wrap up the common bits across the different api calls